woo-viet.php

```diff
@@ -174,6 +171,9 @@ public function main() {
 				include( WOO_VIET_DIR . 'inc/class-wooviet-cities.php' );
 				new WooViet_Cities();
 			}
+
+			// Enqueue province scripts
+			add_action( 'wp_enqueue_scripts', array( $this, 'load_province_scripts' ) );
```

Comment:
Don't do this here. Move all the codes to the Provinces class. 
The main file is not for something like this.